Build on other platforms

When running `ko build --platform=all .`:
```
2023/01/11 09:09:25 Unexpected error running "go build": exit status 1
# github.com/primetalk/goio/io
../../../go/pkg/mod/github.com/primetalk/goio@v0.3.6/io/continuation.go:22:28: cannot use 1000000000000 (untyped int constant) as int value in variable declaration (overflows)
2023/01/11 09:09:31 Unexpected error running "go build": signal: killed
2023/01/11 09:09:31 Unexpected error running "go build": signal: killed
2023/01/11 09:09:31 Unexpected error running "go build": signal: killed
```
on one of the platforms:
```
linux/amd64
linux/arm
linux/ppc64le
linux/s390x
linux/arm64
```

--- a/io/continuation.go
+++ b/io/continuation.go
@@ -18,8 +18,8 @@ type ResultOrContinuation[A any] struct {
 	Continuation *Continuation[A]
 }
 
-// MaxContinuationDepth is equal to 1000000000000. It's the maximum depth we run continuation before giving up.
-var MaxContinuationDepth = 1000000000000
+// MaxContinuationDepth is equal to 1000000000. It's the maximum depth we run continuation before giving up.
+var MaxContinuationDepth = 1000000000
 
 // ObtainResult executes continuation until final result is obtained.
 func ObtainResult[A any](c Continuation[A]) (res A, err error) {

